test/javascripts/turbolinks_test.coffee

```diff
@@ -1,210 +1,946 @@
 describe 'Turbolinks', ->
-
-  createTurboNodule = ->
-    $nodule = $("<div>").attr("id", "turbo-area").attr("refresh", "turbo-area")
-    return $nodule[0]
-
-  url_for = (slug) ->
+  server = null
```

Comment:
Can't run these tests in the browser (`Uncaught ReferenceError: ASSET_FIXTURES is not defined`).  Is it possible to fix this in a later PR?